<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QGroupBox" name="gbController">
      <property name="whatsThis">
       <string>Сканирование IP-контроллера засыланием на него HTTP get запросов</string>
      </property>
      <property name="title">
       <string>IP-контроллер</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="leHostIP">
         <property name="statusTip">
          <string>IP-адрес контроллера</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblIPHost">
         <property name="statusTip">
          <string>IP-адрес контроллера</string>
         </property>
         <property name="text">
          <string>Хост:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="btnScanIP">
         <property name="statusTip">
          <string>Включить\отключить опрос состояния контроллера</string>
         </property>
         <property name="text">
          <string>Сканировать</string>
         </property>
         <property name="icon">
          <iconset resource="resources/ipconserver.qrc">
           <normaloff>:/refresh.png</normaloff>:/refresh.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lblPeriod">
         <property name="statusTip">
          <string>Время периода опроса контроллера</string>
         </property>
         <property name="text">
          <string>Период:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spbIPPeriod">
         <property name="statusTip">
          <string>Время периода опроса контроллера</string>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> * 100 мс</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>299</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spbLampTime">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> с</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="lblLamp">
         <property name="text">
          <string>Время горения лампочки:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lblIPOffPeriod">
         <property name="text">
          <string>Оффлайн. Период опроса:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="spbIPOffPeriod">
         <property name="suffix">
          <string> c</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbShowMessage">
        <property name="statusTip">
         <string>Показывать всплывающие сообщения о сигналах в трее</string>
        </property>
        <property name="text">
         <string>Показывать всплывающие сообщения</string>
        </property>
        <property name="icon">
         <iconset resource="resources/ipconserver.qrc">
          <normaloff>:/chat.png</normaloff>:/chat.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="pteLogIP">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Журнал</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="gbDatabase">
      <property name="whatsThis">
       <string>Подключение к базе данных для добавления записей от срабатываемых сигналов</string>
      </property>
      <property name="title">
       <string>База данных</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lblHostDB">
         <property name="statusTip">
          <string>IP-адрес базы данных</string>
         </property>
         <property name="text">
          <string>Хост:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="leHostDB">
         <property name="statusTip">
          <string>IP-адрес базы данных</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnConnectDB">
         <property name="statusTip">
          <string>Подключиться к базе данных</string>
         </property>
         <property name="text">
          <string>Подключение</string>
         </property>
         <property name="icon">
          <iconset resource="resources/ipconserver.qrc">
           <normaloff>:/map.png</normaloff>:/map.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblRepeat">
         <property name="statusTip">
          <string>В течение этого времени повторные сигналы с одного датчика будут восприниматься как напоминания</string>
         </property>
         <property name="text">
          <string>Запрет повтора:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spbRepeat">
         <property name="statusTip">
          <string>В течение этого времени повторные сигналы с одного датчика будут восприниматься как напоминания</string>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> с</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblDBOffPeriod">
         <property name="text">
          <string>Оффлайн. Период опроса:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spbDBOffPeriod">
         <property name="suffix">
          <string> с</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="gbBrpadcast">
      <property name="whatsThis">
       <string>UDP-рассылка сигналов о срабатывании по сети для клинетов-журналов</string>
      </property>
      <property name="title">
       <string>Броадкаст</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbBCPort">
         <property name="text">
          <string>Порт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spbBCPort">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>45454</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblBCPeriod">
         <property name="text">
          <string>Период:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spbBCPeriod">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> с</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Выход из программы (Alt+F4)</string>
   </property>
  </action>
  <action name="actMinimize">
   <property name="text">
    <string>Свернуть окно</string>
   </property>
  </action>
  <action name="actRestore">
   <property name="text">
    <string>Развернуть окно</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/ipconserver.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>curRecordChanged(QModelIndex)</slot>
 </slots>
</ui>
